xbmc/guilib/guiinfo/GUIInfoLabels.h

```diff
@@ -173,27 +173,28 @@
 #define MUSICPLAYER_RATING          219
 #define MUSICPLAYER_COMMENT         220
 #define MUSICPLAYER_LYRICS          221
-#define MUSICPLAYER_HASPREVIOUS     222
-#define MUSICPLAYER_HASNEXT         223
-#define MUSICPLAYER_EXISTS          224
-#define MUSICPLAYER_PLAYLISTPLAYING 225
-#define MUSICPLAYER_ALBUM_ARTIST    226
-#define MUSICPLAYER_PLAYCOUNT       227
-#define MUSICPLAYER_LASTPLAYED      228
-#define MUSICPLAYER_CHANNEL_NAME    229
-// unused 230
-#define MUSICPLAYER_CHANNEL_GROUP   231
-// unused 232
+#define MUSICPLAYER_ALBUM_ARTIST    222
+#define MUSICPLAYER_PLAYCOUNT       223
+#define MUSICPLAYER_LASTPLAYED      224
+#define MUSICPLAYER_USER_RATING     225
+#define MUSICPLAYER_RATING_AND_VOTES     226
+#define MUSICPLAYER_VOTES           227
+#define MUSICPLAYER_MOOD            228
+#define MUSICPLAYER_CONTRIBUTORS    229
+#define MUSICPLAYER_CONTRIBUTOR_AND_ROLE 230
+#define MUSICPLAYER_CHANNEL_NAME    231
+#define MUSICPLAYER_CHANNEL_GROUP   232
 #define MUSICPLAYER_CHANNEL_NUMBER  233
-#define MUSICPLAYER_CONTENT         234
-#define MUSICPLAYER_USER_RATING     235
-#define MUSICPLAYER_RATING_AND_VOTES     236
-#define MUSICPLAYER_VOTES           237
-#define MUSICPLAYER_MOOD            238
-#define MUSICPLAYER_CONTRIBUTORS    239
-#define MUSICPLAYER_CONTRIBUTOR_AND_ROLE 240
-#define MUSICPLAYER_DBID            241
-#define MUSICPLAYER_PROPERTY        242
+
+#define MUSICPLAYER_DBID            236
+#define MUSICPLAYER_PROPERTY        237
+// Last musicplayer infolabel, check use as limit and amend if new label added
```

Comment:
Why don't you introduce something like MUSICPLAYER_LAST which always points to the last music player label? If a new label gets added, modify MUSICPLAYER_LAST. Use MUSICPLAYER_LAST in MusicGUIInfo. Then you don't need to touch the MusicGUIInfo code every time you add new label.

Reply:
Good idea, but not sure about having such a marker for just those labels that have offset/position rather than all music player things. It starts to seem too bity. I was also wanting to keep this PR to a fix for a backport,

I have WIP that will add more player info labels and will need wider renumbering of defines to make space. It is possibly a step better made then.

Reply:
> Good idea, but not sure about having such a marker for just those labels that have offset/position rather than all music player things.

 I would only add now what is needed to fix this bug. And the LAST define is definitely a good idea to prevent to forget to adjust other code places like in MusicGUIInfo.

 Thinking it big and improving this for all music guiiinfo items is something for another (improvement) PR, imo.

Reply:
Yeap, done